docs(documentation) Add search, theme, pagination, fix sidebar (#494)
johnsonandjohnson/Bodiless-JS: packages/bodiless-documentation/doc/Tools/README.md

# Tools

> This section describes the tools of the BodilessJS system.

Reviewer: ```suggestion
The 'Tools' section contains information on the tools available in BodilessJS.
```
